### PR TITLE
Formatting suggestions for install.md

### PR DESCRIPTION
**Pull request by CBroz1**
Relevant to [your PR here](https://github.com/datajoint/datajoint-elements/pull/42)

### CODE REVIEW

File: install.md [hunk 1]
```diff
@@ -85,20 +82,14 @@ option.
...
-            + Select the [Miniconda installer link](
-                https://conda.io/en/latest/miniconda.html) 
-            for your operating system and following the instructions.
-
-            + You may need to add the Miniconda directory to the PATH environment 
-            variable
-
-                + First locate the Miniconda directory
+        + We will install Miniconda which is a minimal installer for conda.         Follow the [Miniconda installer instructions here]
+        (https://conda.io/en/latest/miniconda.html) for your operating system.
```
Comment (kabilar) [inline install.md]:
```suggestion
        + We will install Miniconda which is a minimal installer for conda.  Follow the [Miniconda installer instructions here](
        https://conda.io/en/latest/miniconda.html) for your operating system.
```

**Review by kabilar — APPROVED** [review]
Thanks @CBroz1.  Just one minor comment to ensure the link formats correctly.

File: install.md [hunk 2]
```diff
@@ -155,146 +146,77 @@ option.
     conda install graphviz python-graphviz pydotplus
```
Comment (CBroz1) [inline install.md:119]: It seems odd to offer both `conda` and `virtualenv` instructions at one stage, and then require `conda` at the next. Should the should the notebook packages section be expanded to include alternatives? Perhaps we should only offer `conda` instructions?

**Comment by CBroz1** [comment]
I updated my copy of `install.md` to be more generic in installation instructions, to avoid adding dedicated sections for `element-trial` and `element-behavior` - @kabilar should I submit a separate PR to the main datajoint account? Future PRs will need to include directory organization and table declaration instructions, when we've made more decisions for the respective repositories.

**Comment by kabilar** [comment]
Thanks @CBroz1.  This is great.  I will review and merge.  No need to submit a separate pull request to the upstream datajoint repository.

File: install.md (hunk 2 above)
Comment (kabilar) [reply install.md:119]: Good point.  For simplicity, let's only include `conda` instructions.  But will make a note that `virtualenv` can also be used.

Comment (kabilar) [reply install.md:119]: I will update my pull request on the base repository.  Thanks @cbroz1.

**Review by kabilar — APPROVED** [review]

**Comment by kabilar** [comment]
Fixed merge conflicts with the above commit.

**Pull request merged**